aws/basic-eks-cluster/xebialabs/xlr-pipeline-ci-cd.yaml.tmpl

```diff
@@ -16,6 +15,36 @@ spec:
     - {{$app}}
     scriptUsername: !value XL_RELEASE_USERNAME
     scriptUserPassword: !value XL_RELEASE_PASSWORD
+
+    variables:     
+      - key: AWSaccessKey
+        type: xlrelease.StringVariable
+        requiresValue: true
+        showOnReleaseStart: false
+        value: {{.AWSAccessKey}}
+      - key: AWSaccessSecret
+        type: xlrelease.StringVariable
+        requiresValue: true
+        showOnReleaseStart: false
+        value: {{.AWSAccessSecret}}
+      - key: AWSRegion
```

Comment:
unnecessary

Reply:
not if the user did not use .aws/credentials